src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,40 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" (empty {}))
+        queue-time (get workflow-counts "estimatedQueueTimeMS" 0)]
+    {:queue-time queue-time
+     :queued (+ (get count-map "Queued" 0) (get count-map "Launching" 0))
+     :active (+ (get count-map "Submitted" 0) (get count-map "Running" 0) (get count-map "Aborting" 0))
+     :terminal (+ (get count-map "Failed" 0) (get count-map "Succeeded" 0) (get count-map "Aborted" 0))}))
+
+(react/defc GlobalSubmissionStatus
+  {:render
+   (fn [{:keys [state]}]
+     (when-let [status-counts (:status-counts @state)]
+       [:span {:style {:marginRight "1ex"}}
+         [:br {}],
+         "Submissions: ", (:queued status-counts), " Queued; ", (:active status-counts), " Active"]))
+   :component-did-mount
+   (fn [{:keys [this locals]}]
+     ;; Call once for initial load
+     (react/call :load-data this)
+     ;; Add a long-polling call for continuous updates
+     (swap! locals assoc :interval-id (js/setInterval #(react/call :load-data this) (config/submission-status-refresh))))
+   :component-will-unmount
+   (fn [{:keys [locals]}]
+     (js/clearInterval (:interval-id @locals)))
+   :load-data
+   (fn [{:keys [state]}]
+     (endpoints/call-ajax-orch
+       {:endpoint (endpoints/submissions-queue-status)
+        :on-done (fn [{:keys [success? status-text get-parsed-response]}]
+          (if success?
+            (swap! state assoc :status-counts (queue-status-counts (get-parsed-response)))
+            ()))}))})
```

Comment:
I struggled with that decision. There isn't really space for a full error, and I'm not sure what to display.